cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/schedule/store/ProgramScheduleStoreDataset.java

```diff
@@ -68,13 +96,22 @@ public ProgramScheduleStoreDataset(DatasetSpecification spec,
   }
 
   /**
+   * Add a schedule to the store.
+   *
+   * @param schedule the schedule to add
+   * @throws AlreadyExistsException if the schedule already exists
+   */
+  public void addSchedule(ProgramSchedule schedule) throws AlreadyExistsException {
+    addSchedules(Collections.singleton(schedule));
+  }
+
+  /**
    * Add one or more schedules to the store.
    *
    * @param schedules the schedules to add
-   *
    * @throws AlreadyExistsException if one of the schedules already exists
    */
-  public void addSchedules(ProgramSchedule... schedules) throws AlreadyExistsException {
+  public void addSchedules(Iterable<ProgramSchedule> schedules) throws AlreadyExistsException {
```

Comment:
seems inconsistent - that the addSchedules method takes Iterable, but the deleteSchedule method takes Collection